addons/google_calendar/tests/test_sync_google2odoo.py

```diff
@@ -975,3 +975,77 @@ def test_recurrence_creation_with_attendee_answer(self):
         attendee = recurrence.calendar_event_ids.attendee_ids.mapped('state')
         self.assertEqual(attendee, ['declined', 'declined', 'declined'], "All events should be declined")
         self.assertGoogleAPINotCalled()
+
+    @patch_api
+    def test_several_attendee_have_the_same_mail(self):
+        """In google, One mail = One attendee but on Odoo, some partners could share the same mail
+        This test check that the deletion of such attendee has no harm: all attendee buth the given mail are deleted.
```

Comment:
```suggestion

        This test check that the deletion of such attendee has no harm: all attendee buth the given mail are deleted.
```
Sphinx docstrings need a blank line between the title and the body. Same as for commits.

Reply:
```suggestion
        This test checks that the deletion of such attendee has no harm: all attendees but the given mail are deleted.
```